### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by creydr**
Same as #3503 but fixes the build issues

/cc @Cali0707 @matzew

### CODE REVIEW

**Review by matzew — APPROVED** [review]
/lgtm
/approve

File: test/e2e_new/new_trigger_filters_test.go [hunk 1]
```diff
@@ -44,10 +44,8 @@ func TestNewTriggerFilters(t *testing.T) {
 		k8s.WithEventListener,
 		environment.Managed(t),
 	)
-	brokerName := feature.MakeRandomK8sName("broker")
 
-	env.Prerequisite(ctx, t, InstallKafkaBroker(brokerName))
-	env.TestSet(ctx, t, newfilters.FiltersFeatureSet(brokerName))
+	env.TestSet(ctx, t, newfilters.NewFiltersFeatureSet(InstallKafkaBroker))
```
Comment (Cali0707) [inline test/e2e_new/new_trigger_filters_test.go:48]: Hey @creydr the change which caused the build issuesactually replaced all these individual test calls into one big ParallalTestSet, see here: https://github.com/knative/eventing/pull/7424/files#diff-8ce90a262068afe6d5d4ca677978d7e7ed9af0acab94ef93424bd6b78b535f18

**Pull request merged**